html2asketch/rectangle.js

```diff
@@ -73,6 +79,7 @@ class Rectangle extends Base {
     obj.fixedRadius = 0;
     obj.edited = false;
     obj.booleanOperation = -1;
+    obj.shouldBreakMaskChain = this._shouldBreakMaskChain;
```

Comment:
Maybe this configuration should be set within Base instead of adding the logic to `Text` and `Rectangle` (and potentially more subclasses moving forward)

Reply:
Good point, lets add move it to Base.